test/test_encryption.py

```diff
@@ -2997,5 +2999,58 @@ def test_collection_name_collision(self):
         self.assertIsInstance(exc.exception.encrypted_fields["fields"][0]["keyId"], Binary)
 
 
+def start_mongocryptd(port) -> None:
+    args = ["mongocryptd", f"--port={port}", "--idleShutdownTimeoutSecs=60"]
+    _spawn_daemon(args)
+
+
+class TestNoSessionsSupport(EncryptionIntegrationTest):
+    mongocryptd_client: MongoClient
+    MONGOCRYPTD_PORT = 27020
+
+    @classmethod
+    @client_context.require_sessions
```

Comment:
This test is for servers that _don't_ support sessions, but require_sessions does the opposite. Can we remove this?